aria/modeling/service_template.py

```diff
@@ -1848,11 +1856,30 @@ def as_raw(self):
 
     def instantiate(self, container):
         from . import models
+        from ..orchestrator import context
+        plugin = None
+        if self.plugin_specification is not None:
```

Comment:
You are correct, I had a long conversation with Maxim about it. Eventually when we have an instantiation module, it is very likely that the module will have a context. See comment at line 1861.